hugegraph-core/src/main/java/org/apache/hugegraph/StandardHugeGraph.java

```diff
@@ -1063,6 +1063,10 @@ public String toString() {
         return StringFactory.graphString(this, this.name());
     }
 
+    public HugeGraphParams hugeGraphParams() {
```

Comment:
it's not allowed to pass HugeGraphParams out, we need to prevent the underlying method from being called by gremlin without authorization.